### ISSUE TITLE
Cannot GET /about

### ISSUE THREAD

**Issue by Artur-** [vaadin/studio#1]
### Steps to reproduce
```
vaadin init studio-test --latest --fusion --git
```

Open live view for `main-view.ts`. Click on "About" in the menu.

### Actual behavior
Live view goes white.
Selecting another TS file shows "Cannot GET /about"

### Expected behavior
You select the anchor and the location is shown in the code

### Environment, such as OS version, node version, Java version, VS Code and Studio versions
macOS
Studio 0.1.0

**Comment by gilberto-torrezan** [comment]
Yes, this happens because In the Live View there's no support for navigation yet - the "About" link in the menu is meant to navigate to another view.

In the future Live View will have two modes of operation: one that all UI interactions are blocked (the preview mode), and clicking around triggers the selection in the code; and the second mode where the application behaves as the real thing (the app runtime mode), and clicking in navigation links should get to the proper view.

**Comment by Artur-** [comment]
I would absolutely not expect it to navigate when I try to edit my link
